#! /bin/sh -e
## ubuntu-default-error-policy-retry-job.dpatch by  <martin.pitt@ubuntu.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Set default job error policy to "Retry", since it is less confusing and
## DP: a better default on desktop machines. This patch is only applied when
## DP: building on Ubuntu.

[ "`lsb_release -is 2>/dev/null`" = "Ubuntu" ] || { echo -n '(ignored)' >&2; exit 0; }
. /usr/share/dpatch/dpatch-run

@DPATCH@
diff -urNad cups-1.3.0~rc2~/scheduler/conf.c cups-1.3.0~rc2/scheduler/conf.c
--- cups-1.3.0~rc2~/scheduler/conf.c	2007-07-30 17:47:10.000000000 +0100
+++ cups-1.3.0~rc2/scheduler/conf.c	2007-08-01 15:33:55.000000000 +0100
@@ -548,7 +548,7 @@
   cupsdClearString(&BrowseLocalOptions);
   cupsdClearString(&BrowseRemoteOptions);
 
-  cupsdSetString(&ErrorPolicy, "stop-printer");
+  cupsdSetString(&ErrorPolicy, "retry-job");
 
 #ifdef HAVE_LDAP
   cupsdClearString(&BrowseLDAPBindDN);
